src/audio/coreaudio/SDL_coreaudio.m

```diff
@@ -901,15 +903,13 @@ static BOOL update_audio_session(_THIS, SDL_bool open, SDL_bool allow_playandrec
     }
 
     /* Make sure we can feed the device a minimum amount of time */
-    double MINIMUM_AUDIO_BUFFER_TIME_MS = 15.0;
+    MINIMUM_AUDIO_BUFFER_TIME_MS = 15.0;
```

Comment:
I can initialise it at the top, but it can't be set to const as it *may* be mutated on iOS right below where it was initially declared:
```c
    /* Make sure we can feed the device a minimum amount of time */
    MINIMUM_AUDIO_BUFFER_TIME_MS = 15.0;
#if defined(__IPHONEOS__)
    if (floor(NSFoundationVersionNumber) <= NSFoundationVersionNumber_iOS_7_1) {
        /* Older iOS hardware, use 40 ms as a minimum time */
        MINIMUM_AUDIO_BUFFER_TIME_MS = 40.0;
    }
#endif
```

Would you like to me to move the initialisation anyway ?

Reply:
Ah, I see. No, that's fine.